tokio/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 0.2.19 (April 24, 2020)
+
+### Fixes
+- docs: misc improvements (#2400, #2414, #2420, #2423, #2405, #2427, #2426, #2434, #2436, #2440).
```

Comment:
Can we sort these? :sweat_smile: 
```suggestion
- docs: misc improvements (#2400, #2405, #2414, #2420, #2423, #2426, #2427, #2434, #2436, #2440).
```

Reply:
I pretty much just go up the changelog from old to new :shrug: